lib/archive_importer/blocks_importer.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class ArchiveImporter
+  class BlocksImporter
+    include Diaspora::Logging
+  
+    def initialize(json, user)
+      @json = json
+      @user = user
+    end
+
+    attr_reader :json
+    attr_reader :user
+
+    def import
+      p = Person.find_or_fetch_by_identifier(json)
+      migrant_person = handle_migrant_person(p)
+      user.blocks.create(person_id: migrant_person.id) if migrant_person.present?
+    rescue ActiveRecord::RecordInvalid,
+            DiasporaFederation::Discovery::DiscoveryError => e
```

Comment:
Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.